vnext/ReactWindows-Desktop.sln

```diff
@@ -81,7 +71,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "DynamicLibrary", "DynamicLi
 		PropertySheets\DynamicLibrary\x86.props = PropertySheets\DynamicLibrary\x86.props
 	EndProjectSection
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Shared", "Shared\Shared.vcxitems", "{2049DBE9-8D13-42C9-AE4B-413AE38FFFD0}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Microsoft.React.Shared", "Shared\Shared.vcxitems", "{2049DBE9-8D13-42C9-AE4B-413AE38FFFD0}"
```

Comment:
`Microsoft.ReactNative.Shared` here and elsewhere (rename Shared.vcxitems too?)

Reply:
There's no need to rename the VCXITEMS file, nor directory, as long as the `ProjectName` or `ItemsProjectName` property is set correctly.